<?xml version="1.0"?>


<launch>
	<include file="$(find andbot)/launch/display.launch" />
<!--	
	<include file="$(find andbot)/launch/fake_move_base.launch" />
-->

	<include file="$(find andbot)/launch/fake_move_base_blank_map.launch" />

<!--	
	<include file="$(find andbot)/launch/xv_11.launch" />

	<node pkg="andbot" type="mybot_xv11_angular_bound_filter" name="mybot_xv11_angular_bound_filter" />
	
	<node pkg="andbot" type="mybot_laser_obstacle_clearing_filter" name="mybot_laser_obstacle_clearing_filter" />
-->	
	<node pkg="andbot" type="mybot_base_controller" name="mybot_base_controller" />	
	<!-- run serial node for base mega -->	
	<node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    		<param name="port" value="/dev/mega_base0"/>
    		<param name="baud" value="57600"/>
  	</node>
	
	<node pkg="andbot" type="mybot_odometry" name="mybot_odometry" />
<!--
	<include file="$(find andbot)/launch/gmapping.launch" />
-->
<!--
	<node pkg="rosserial_python" type="serial_node1.py" name="serial_node1">
                <param name="port" value="/dev/ttyACM1"/>
                <param name="baud" value="57600"/>
        </node>

	<node pkg="rosserial_python" type="serial_node2.py" name="serial_node2">
                <param name="port" value="/dev/ttyACM2"/>
                <param name="baud" value="57600"/>
        </node>
-->

<!--
	<include file="$(find andbot)/launch/robot_mapping.launch" />
-->
</launch>
